client/landing/jetpack-cloud/components/backup-delta/index.jsx

```diff
@@ -35,62 +40,180 @@ class BackupDelta extends Component {
 		return (
 			<div className="backup-delta__realtime">
 				<div>{ translate( 'More backups from today' ) }</div>
-				{ cards.length ? cards : <div>{ translate( 'you have no more backups for this day' ) }</div> }
+				{ cards.length ? (
+					cards
+				) : (
+					<div>{ translate( 'you have no more backups for this day' ) }</div>
+				) }
 			</div>
 		);
 	}
 
+	renderMetaDiff() {
+		const { metaDiff } = this.props;
+		const metas = [];
+
+		metaDiff.forEach( meta => {
+			if ( meta.num > 0 || meta.num < 0 ) {
+				const operator = meta.num < 0 ? '' : '+';
+				const plural = meta.num > 1 || meta.num < -1 ? 's' : '';
+				// TBD: How do we deal with translating these strings?
+				metas.push( `${ operator }${ meta.num } ${ meta.type }${ plural }` );
+			}
+		} );
+
+		return <div className="backup-delta__metas">{ metas.join( ', ' ) }</div>;
+	}
+
 	renderDaily() {
-		const { backupAttempts, deltas, siteSlug, translate } = this.props;
+		const { backupAttempts, deltas, metaDiff, siteSlug, translate } = this.props;
 		const mainBackup = backupAttempts.complete && backupAttempts.complete[ 0 ];
-		const meta = mainBackup && mainBackup.activityDescription[ 2 ].children[ 0 ];
 
-		const media = deltas.mediaCreated.map( item => (
-			<div key={ item.activityId }>
-				<img alt="" src={ item.activityMedia.thumbnail_url } />
-				<div>{ item.activityMedia.name }</div>
+		const mediaCreated = deltas.mediaCreated.map( item => (
+			<div key={ item.activityId } className="backup-delta__media-image">
+				<img
+					alt=""
+					src={ item.activityMedia.available ? item.activityMedia.thumbnail_url : mediaImage }
+				/>
+				<div className="backup-delta__media-title">
+					<Gridicon icon="plus" />
+					<div className="backup-delta__media-title-text">{ translate( 'Added' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **81** times:
`translate( 'Added:' )` *ES Score: 7*